invenio_orcid/config.py

```diff
@@ -26,6 +26,12 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_AUTHORS_SEARCH_CLASS = "invenio_search:RecordsSearch"
+
+ORCID_RECORDS_PID_TYPE = 'literature'
+ORCID_RECORDS_DOC_TYPE = 'hep'
```

Comment:
I think we can put literature here as well, since `hep` it's really INSPIRE specific.